site-src/v1alpha2/references/policy-attachment.md

```diff
@@ -0,0 +1,294 @@
+
+At the other extreme, policy can be used to apply to requests from a specific
+workload to a backend in another namespace. A route can be used to intercept
+these requests and split them between different backends (foo-a and foo-b in
+this case).
+
+![Complex Mesh Example](/images/policy/mesh-complex.png)
+
+## Target Reference API
+
+Each Policy resource MUST include a single `targetRef` field. It MUST not
+target more than one resource at a time, but it can be used to target larger
+resources such as Gateways or Namespaces that may apply to multiple child
+resources.
+
+The `targetRef` field MUST be an exact replica of the `PolicyTargetReference`
+struct included in the Gateway API. Where possible, it is recommended to use
+that struct directly instead of duplicating the type.
+
+### Policy Boilerplate
+The following structure MUST be used as for any Policy resource using this API
+pattern. Within the spec, policy resources may omit `Override` or `Default`
+fields, but at least one of them MUST be present.
+
+```go
+// ACMEServicePolicy provides a way to apply Service policy configuration with
+// the ACME implementation of the Gateway API.
+type ACMEServicePolicy struct {
+    metav1.TypeMeta   `json:",inline"`
+    metav1.ObjectMeta `json:"metadata,omitempty"`
+
+    // Spec defines the desired state of ACMEServicePolicy.
+    Spec ACMEServicePolicySpec `json:"spec"`
+
+    // Status defines the current state of ACMEServicePolicy.
+    Status ACMEServicePolicyStatus `json:"status,omitempty"`
+}
+
+// ACMEServicePolicySpec defines the desired state of ACMEServicePolicy.
+type ACMEServicePolicySpec struct {
+    // TargetRef identifies an API object to apply policy to.
+    TargetRef gatewayv1a2.PolicyTargetReference `json:"targetRef"`
+
+    // Override defines policy configuration that should override policy
+    // configuration attached below the targeted resource in the hierarchy.
+    // +optional
+    Override *ACMEPolicyConfig `json:"override,omitempty"`
+
+    // Default defines default policy configuration for the targeted resource.
+    // +optional
+    Default *ACMEPolicyConfig `json:"default,omitempty"`
+}
+
+// ACMEPolicyConfig contains ACME policy configuration.
+type ACMEPolicyConfig struct {
+    // Add configurable policy here
+}
+
+// ACMEServicePolicyStatus defines the observed state of ACMEServicePolicy.
+type ACMEServicePolicyStatus struct {
+    // Conditions describe the current conditions of the ACMEServicePolicy.
+    //
+    // +optional
+    // +listType=map
+    // +listMapKey=type
+    // +kubebuilder:validation:MaxItems=8
+    Conditions []metav1.Condition `json:"conditions,omitempty"`
+}
+```
+
+### Hierarchy
+Each policy MAY include default or override values. Overrides enable admins to
+enforce policy from the top down. Defaults enable app owners to provide default
+values from the bottom up for each individual application.
+
+![Policy Hierarchy](/images/policy/hierarchy.png)
```

Comment:
Replacing sidecar with mesh in this diagram might be for the best.
Mesh is a concept. Sidecar is a deployment paradigm.

Reply:
Updated, thanks!